### DISCUSSION TITLE
Transparent Roadmap

### DISCUSSION THREAD

**Discussion by JtheBAB** [mattermost/community#152] [Practical Ideas]
It would be really awesome to have a channel on community where you can see a monthly report in which state, which project like CRT or Custom Status is. 

That would help that everyone is sync about the product. It helps also to understand where the route goes and what can be expected.

**Comment by esethna** [comment]
Yes we're working on that to get our internal tool (Productboard) to be more public so you can see what is coming next month, 3 months, 6 months and later
